src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -88,6 +88,9 @@ internal static class IDEDiagnosticIds
 
         public const string FormattingDiagnosticId = "IDE0055";
 
+        public const string UseIndexOperatorDiagnosticId = "IDE0055";
```

Comment:
This conflicts with `FormattingDiagnosticId` above.

Reply:
Great catch.  Yaay cross branch merges.